release/air_tests/air_benchmarks/workloads/xgboost_benchmark.py

```diff
@@ -102,7 +102,7 @@ def run_xgboost_prediction(model_path: str, data_path: str):
     ds = data.read_parquet(data_path)
     ckpt = XGBoostCheckpoint.from_model(booster=model)
     batch_predictor = BatchPredictor.from_checkpoint(ckpt, XGBoostPredictor)
-    result = batch_predictor.predict(ds.drop_columns(["labels"]))
+    result = batch_predictor.predict(ds.drop_columns(["labels"]), batch_size=8192)
```

Comment:
shall we also add a comment? Thanks.

Reply:
sure i can add one once release test on this PR finish